ui/src/components/OverviewTable.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Table, TableHeader, TableBody, sortable, SortByDirection } from '@patternfly/react-table';
+import { Redirect } from 'react-router-dom';
+
+class OverviewTable extends React.Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      columns: [
+        { title: 'Apps', transforms: [ sortable ] },
+        { title: 'Num of Deployed Versions', transforms: [ sortable ] },
+        { title: 'Num of Clients', transforms: [ sortable ] },
+        { title: 'Numb of App Sartups', transforms: [ sortable ] }
+      ],
+      rows: [
+        [ 'App-A', 3, 245, 4 ],
+        [ 'App-B', 4, 655, 5 ],
+        [ 'App-C', 1, 970, 6 ],
+        [ 'App-D', 6, 255, 7 ],
+        [ 'App-E', 5, 120, 8 ]
+      ],
+
+      sortBy: {},
+      redirect: false
+    };
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+  }
+
+  onRowClick (event, rowId, props) {
+    console.log('navigate was called on row: ', rowId.apps.title);
+    this.setState({ redirect: true });
+  }
+
+  onSort (_event, index, direction) {
```

Comment:
Is this built into PF4 already? Rather than us having to add the code.

Reply:
I'll have a look into it. Some of these methods are provided as overrides but I'm not sure about this one.

Reply:
Yes we need to have this method to do the sorting. The onSort should be managed by Redux state. This is being address in [this pr](https://github.com/aerogear/mobile-security-service/pull/68), so resolving this conversation.